knexfile.js

```diff
@@ -1,4 +1,5 @@
 // Update with your config settings.
```

Comment:
Can you delete this comment right here? I think I might have accidentally put this comment here.

Reply:
Done.